packages/regl-worldview/src/commands/Command.js

```diff
@@ -84,46 +84,26 @@ export default class Command<T> extends React.Component<Props<T>> {
     if (!context) {
       return;
     }
-    const drawProps = this.props.drawProps;
-    if (drawProps == null) {
+
+    const { children, reglCommand, layerIndex, getHitmap } = this.props;
+    if (children == null) {
       return;
     }
     context.registerDrawCall({
       instance: this,
-      command: this.props.reglCommand,
-      drawProps,
-      layerIndex: this.props.layerIndex,
+      command: reglCommand,
```

Comment:
nit: maybe rename this to reglCommand since all the others are keeping their original names too